examples/some-verifier/frontend/src/scss/layout/_root.scss

```diff
@@ -0,0 +1,9 @@
+html {
+  height: 100%;
+}
+
```

Comment:
Suggestion: Starting the front end at porst `8080` instead of `80` by default would avoid access permission problems.

Reply:
Right. Port 8080 is already used as default for the telegram issuer, along with 8081 for discord.

I think it's a good point, but will keep it as is for now as it's not really related to this specifically, and I cannot remember the implications of changing the port with regards to different services depending on it, oAuth stuff, etc.

Also it works on mac without any issues, so maybe get another OS? 😁